<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="add">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>80</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить/
Изменить</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="station">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Метеостанция</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="vane">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>40</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Флюгер</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Delete_all">
    <property name="geometry">
     <rect>
      <x>642</x>
      <y>359</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить всё</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Weather">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>170</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Погода</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="row">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>20</y>
      <width>51</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="col">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>20</y>
      <width>51</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="row_text">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>20</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>row</string>
    </property>
   </widget>
   <widget class="QLabel" name="col_text">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>20</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>col</string>
    </property>
   </widget>
   <widget class="QLabel" name="col_text_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>170</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>col</string>
    </property>
   </widget>
   <widget class="QLabel" name="row_text_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>170</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>row</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="col_weather">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>170</y>
      <width>51</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="row_weather">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>170</y>
      <width>51</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Load">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>260</y>
      <width>151</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Загрузить из файла</string>
    </property>
   </widget>
   <widget class="QLabel" name="weather_text">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>210</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Save">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>260</y>
      <width>151</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Загрузить в файл</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="path">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>300</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="pres">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>60</y>
      <width>51</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="temp">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>60</y>
      <width>51</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="dorection">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>100</y>
      <width>51</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="speed">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>100</y>
      <width>51</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="row_text_3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>60</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>temp</string>
    </property>
   </widget>
   <widget class="QLabel" name="row_text_4">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>60</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>pres</string>
    </property>
   </widget>
   <widget class="QLabel" name="row_text_5">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>100</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>WS</string>
    </property>
   </widget>
   <widget class="QLabel" name="row_text_6">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>100</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>WD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setSize">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Установить размер</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sizeSpin">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>350</y>
      <width>48</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>391</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>769</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
